__tests__/buildx.test.ts

```diff
@@ -28,6 +28,34 @@ describe('isAvailable', () => {
   });
 });
 
+describe('getRelease', () => {
```

Comment:
Would be nice to have a comment in this test which links to the upstream file in GitHub, to easily find it if this test breaks :tada:

Reply:
Our actions-toolkit will handle that directly in a follow-ups :pray: